packages/client/src/MWPClient.ts

```diff
@@ -49,6 +49,9 @@ export class MWPClient {
       ...metadata,
       appName: metadata.appName || 'Dapp',
       appDeeplinkUrl: appendMWPResponsePath(metadata.appDeeplinkUrl),
+      appCustomScheme: metadata.appCustomScheme
+        ? appendMWPResponsePath(metadata.appCustomScheme)
```

Comment:
@fan-zhang-sv We might need to update `appendMWPResponsePath` to handle custom schemes. `url.pathname` doesn't seem to work the same way as for URLs.
<img width="387" alt="image" src="https://github.com/user-attachments/assets/b12290f4-d266-4b31-ba63-2769d03a0859">

Reply:
Confirmed offline that pathname does get appended in React Native environment